src/styles/utils/_extend.scss

```
@@ -0,0 +1,7 @@
+%size {
+  display: flex;
+  position: absolute;
+  width: $width-arrow;
+  border-radius: 100%;
```

```suggestion
  border-radius: 50%;
```
цього достатньо для кола

---

Також демо не працює

---

> Також демо не працює

демо чомусь і не працює, балакали про це в Slack з [marina-tilniak](https://github.com/marina-tilniak)  вона казала просто виправити помилки.
https://mate-parttime.slack.com/archives/C03RJ1B6EN4/p1665504499107459